app/client/src/ce/pages/Applications/index.tsx

```
@@ -134,12 +133,26 @@ import { LayoutSystemTypes } from "layoutSystems/types";
   const isGitModEnabled = useGitModEnabled();
 
-  return isGitModEnabled ? <NewGitImportModal /> : <OldGitSyncModal isImport />;
+  return isGitModEnabled ? (
+    <>
+      <NewGitImportModal />
+      <NewGitRepoLimitErrorModal />
+    </>
+  ) : (
+    <>
+      <OldGitSyncModal isImport />;
+      <OldRepoLimitExceededErrorModal />
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)